packages/domain/lib/usecases/user_is_registered_usecase.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:domain/entities/user_entity.dart';
+import 'package:domain/repositories/remote_store_repository.dart';
+import 'package:domain/usecases/usecase.dart';
+
+class UserIsRegisteredUseCase
+    implements UseCaseParams<UserEntity, Future<bool>> {
+  final RemoteStoreRepository _storeRepository;
+
+  UserIsRegisteredUseCase(this._storeRepository);
+
+  @override
+  Future<bool> call(UserEntity user) async {
+    final userDocData = await _storeRepository.getDocumentDataByValues(
+      "users", // TODO move to constant
```

Comment:
move to constant

Reply:
Was fixed in penult commit, possible you saw this TODO seeing some of initial commits.